packages/admin-stories/src/docs/components/ColorPicker/stories/ColorPaletteAndPicker.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import { Field } from "@comet/admin";
+import { FinalFormColorPicker } from "@comet/admin-color-picker";
+import { Favorite, Warning } from "@comet/admin-icons";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import { Form } from "react-final-form";
+
+storiesOf("stories/components/Color Picker/Color Palette and Color Picker", module)
+    .add("Color Palette and Color Picker", () => {
```

Comment:
I'm concerned that defining multiple stories in a single file might decrease the readability of the stories... What dou you think?

Reply:
you mean that for developing the stories or for viewing the stories?

Reply:
For developing

Reply:
yes, can be overwhelming. but there is no technical reason for that, right?

Reply:
No, there isn't. If we're okay with it, we can keep it like it is now 🙂

Reply:
@johnnyomair thought about the same thing. 

I Changed this, so we have separat files for each story. The downside of this is that we need unique paths in `storiesOf()` in separat files. otherwise the source code loader of storybook does not work correctly, and we get "No Code available" below stories.